WMass/python/plotter/w-helicity-13TeV/plotYW.py

```diff
@@ -68,11 +71,13 @@ def makeGraphs(self):
     def makeMultiGraphRel(self):
         self.mg = ROOT.TMultiGraph()
         self.mg.SetTitle() ## no title 'W^{{{ch}}}: {p}'.format(ch=self.ch,p=self.pol))
-        #self.shiftPoints(self.graph_fit_rel)
+        self.shiftPoints(self.graph_fit_rel)
```

Comment:
comment out again the shiftPoints. Since we plot the ratios in separate pads for L and R, no need to shift and it is confusing